activerecord/lib/active_record/counter_cache.rb

```diff
@@ -73,13 +89,34 @@ def reset_counters(id, *counters)
       #   # UPDATE posts
       #   #    SET comment_count = COALESCE(comment_count, 0) + 1
       #   #  WHERE id IN (10, 15)
+      #
+      #   # For the Posts with id of 10 and 15, increment the comment_count by 1
+      #   # and update the updated_at value for each counter.
+      #   Post.update_counters [10, 15], comment_count: 1, touch: true
```

Comment:
ActiveRecord::Callbacks defines [#touch](https://github.com/rails/rails/blob/18325d733360fc864eb800d95e56f404dfeeb7d6/activerecord/lib/active_record/callbacks.rb#L280-L282), so if you have a column named `touch` then it will raise `ActiveRecord::DangerousAttributeError`.